[2025-05-06T00:28:48.386+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-05-06T00:28:48.398+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dbt_sequential_run.dbt_run_intermediate manual__2025-05-06T00:04:35.922172+00:00 [queued]>
[2025-05-06T00:28:48.405+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dbt_sequential_run.dbt_run_intermediate manual__2025-05-06T00:04:35.922172+00:00 [queued]>
[2025-05-06T00:28:48.406+0000] {taskinstance.py:2865} INFO - Starting attempt 2 of 3
[2025-05-06T00:28:48.415+0000] {taskinstance.py:2888} INFO - Executing <Task(BashOperator): dbt_run_intermediate> on 2025-05-06 00:04:35.922172+00:00
[2025-05-06T00:28:48.423+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=3189) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-05-06T00:28:48.422+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'dbt_sequential_run', 'dbt_run_intermediate', 'manual__2025-05-06T00:04:35.922172+00:00', '--job-id', '57', '--raw', '--subdir', 'DAGS_FOLDER/dbt_run_dag.py', '--cfg-path', '/tmp/tmp51aq5c5v']
[2025-05-06T00:28:48.424+0000] {standard_task_runner.py:72} INFO - Started process 3190 to run task
[2025-05-06T00:28:48.424+0000] {standard_task_runner.py:105} INFO - Job 57: Subtask dbt_run_intermediate
[2025-05-06T00:28:48.460+0000] {task_command.py:467} INFO - Running <TaskInstance: dbt_sequential_run.dbt_run_intermediate manual__2025-05-06T00:04:35.922172+00:00 [running]> on host 5f6a5c55bc08
[2025-05-06T00:28:48.545+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='dbt_sequential_run' AIRFLOW_CTX_TASK_ID='dbt_run_intermediate' AIRFLOW_CTX_EXECUTION_DATE='2025-05-06T00:04:35.922172+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-05-06T00:04:35.922172+00:00'
[2025-05-06T00:28:48.548+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-05-06T00:28:48.565+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2025-05-06T00:28:48.569+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', '/home/***/.local/bin/dbt run --profiles-dir /opt/***/dbt --project-dir /opt/***/dbt --select intermediate']
[2025-05-06T00:28:48.576+0000] {subprocess.py:86} INFO - Output:
[2025-05-06T00:28:50.881+0000] {subprocess.py:93} INFO - [0m00:28:50  Running with dbt=1.8.7
[2025-05-06T00:28:52.066+0000] {subprocess.py:93} INFO - [0m00:28:52  Registered adapter: snowflake=1.8.1
[2025-05-06T00:28:54.530+0000] {subprocess.py:93} INFO - [0m00:28:54  Found 12 models, 10 data tests, 7 sources, 459 macros
[2025-05-06T00:28:54.532+0000] {subprocess.py:93} INFO - [0m00:28:54
[2025-05-06T00:28:56.196+0000] {subprocess.py:93} INFO - [0m00:28:56  Concurrency: 4 threads (target='dev')
[2025-05-06T00:28:56.207+0000] {subprocess.py:93} INFO - [0m00:28:56
[2025-05-06T00:28:56.232+0000] {subprocess.py:93} INFO - [0m00:28:56  1 of 1 START sql view model RAW_public.int_customer_enriched ................... [RUN]
[2025-05-06T00:28:57.544+0000] {subprocess.py:93} INFO - [0m00:28:57  1 of 1 OK created sql view model RAW_public.int_customer_enriched .............. [[32mSUCCESS 1[0m in 1.30s]
[2025-05-06T00:28:57.555+0000] {subprocess.py:93} INFO - [0m00:28:57
[2025-05-06T00:28:57.556+0000] {subprocess.py:93} INFO - [0m00:28:57  Finished running 1 view model in 0 hours 0 minutes and 3.02 seconds (3.02s).
[2025-05-06T00:28:57.704+0000] {subprocess.py:93} INFO - [0m00:28:57
[2025-05-06T00:28:57.705+0000] {subprocess.py:93} INFO - [0m00:28:57  [32mCompleted successfully[0m
[2025-05-06T00:28:57.706+0000] {subprocess.py:93} INFO - [0m00:28:57
[2025-05-06T00:28:57.707+0000] {subprocess.py:93} INFO - [0m00:28:57  Done. PASS=1 WARN=0 ERROR=0 SKIP=0 TOTAL=1
[2025-05-06T00:28:58.508+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2025-05-06T00:28:58.523+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-05-06T00:28:58.524+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=dbt_sequential_run, task_id=dbt_run_intermediate, run_id=manual__2025-05-06T00:04:35.922172+00:00, execution_date=20250506T000435, start_date=20250506T002848, end_date=20250506T002858
[2025-05-06T00:28:58.538+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-05-06T00:28:58.555+0000] {taskinstance.py:3900} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-05-06T00:28:58.556+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
